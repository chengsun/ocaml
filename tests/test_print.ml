let _ = print_string "static"; print_newline ()
let _ = print_int 1; print_newline ()
let _ = print_float 3.141592653; print_newline ()
