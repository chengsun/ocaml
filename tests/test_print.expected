static
1
3.141592653
